### ISSUE TITLE
Pair Match scroll bar not working

### ISSUE THREAD

**Issue by scholar6admin** [Scholar-6/brillder#4490]
![image](https://user-images.githubusercontent.com/58778078/162424214-ba5ea859-b77d-4269-bd9c-6f99a1558b1f.png)

**Comment by s-steddy** [comment]
Brick is 'Adjectives with 'ser' and 'estar''

[x] If issue was that pair-match could not be changed, this seems to have been fixed.

Something very odd going on with this brick. Images in synthesis are buggy, and stay up on the screen. If one is open when getting to review, it will get stuck. This is from the end of the synthesis and persisted as I clicked through questions in review. (Tested with clean install of chrome).

<img width="962" alt="Screenshot 2022-08-01 at 16 28 11" src="https://user-images.githubusercontent.com/107257153/182185842-b9ffdec9-d040-4b3e-bd2e-f3753acac8bd.png">

Looking on other bricks, images in general seem quite buggy, the hover-over zoom is unpredictable in particular: hover-zoom will open when mouse is not on image, and will not unzoom when mouse moves away. Will open new issue once I can define the problem a bit better, but issue with this brick seems to need a particular look.
